src/deploy/D3MInit.sol

```diff
@@ -16,7 +16,11 @@
 
 pragma solidity >=0.8.0;
 
-import "dss-interfaces/Interfaces.sol";
+import {
```

Comment:
I don't think this makes a change for the flatten script, it will still load everything.  We should be doing instead:
```
import "dss-interfaces/dss/DssAutoLineAbstract.sol";
import "dss-interfaces/dss/IlkRegistryAbstract.sol";
import "dss-interfaces/ERC/GemAbstract.sol";
```

Reply:
done